make/release.mk

```diff
@@ -4,9 +4,14 @@ S3_ACL ?= "bucket-owner-full-control"
 
 .PHONY: release
 release: ARCHIVE_NAME = kommander-applications-$(GIT_TAG).tar.gz
+release: PUBLISHED_URL = https://downloads.d2iq.com/dkp/$(GIT_TAG)/kommander-applications-$(GIT_TAG).tar.gz
```

Comment:
```suggestion
release: PUBLISHED_URL = https://downloads.d2iq.com/dkp/$(GIT_TAG)/$(ARCHIVE_NAME)
```

Reply:
In this case, the curl was complaining about the invalid payload since `ARCHIVE_NAME` is itself using `GIT_TAG`. That's why I kept it like that.